src/feature/flow_chart_panel/FlowChartKeyboardShortcuts.tsx

```diff
@@ -0,0 +1,94 @@
+import ReactFlow, {
```

Comment:
Unnecessarily long file, no need to wrap the `ReactFlow` instance with this component. You could just keep `ReactFlow` component in `FlowChartTabView` as it was before, and just make a component that only calls hooks and returns an empty div, which avoids this long prop drilling and is significantly shorter.
```typescript
const FlowChartKeyboardShortcuts = () => {
  const { zoomIn, zoomOut, fitView } = useReactFlow();
  useKeyboardShortcut("ctrl", "=", zoomIn);
  useKeyboardShortcut("ctrl", "-", zoomOut);
  useKeyboardShortcut("ctrl", "1", fitView);

  useKeyboardShortcut("meta", "=", zoomIn);
  useKeyboardShortcut("meta", "-", zoomOut);
  useKeyboardShortcut("meta", "1", fitView);

  return <div />;
}
```